newsrevenuehub/rev-engine
# Dev 1111 contributor portal rp branded

Please fill out each section even if it's just with "N/A"

#### Plan? (if this draft/incomplete indicate what you intend to do and how)
[DEV-1111]

#### What's this PR do?
provides custom-styling based on default page for an RP

#### Why are we doing this? How does it help us?
to make contributor dashboard similar to styling used by RP

#### How should this be manually tested?
Check instructions at link below
https://news-revenue-hub.atlassian.net/browse/DEV-1111?focusedCommentId=44443

#### Have automated unit tests been added?
no

#### How should this change be communicated to end users?
n/a

#### Are there any smells or added technical debt to note?
no

#### Has this been documented? If so, where?
no

#### What are the relevant tickets?
**(Note: Please use syntax [JIRA-123] to link any relevant tickets)**
[dev-1111]

#### Do any changes need to be made before deployment to production (adding environment variables, for example)?
n/a

#### Are there next steps? If so, what? Have tickets been opened for them?



[DEV-1111]: https://news-revenue-hub.atlassian.net/browse/DEV-1111?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
